packages/kit/src/runtime/server/respond.js

```diff
@@ -573,6 +573,13 @@ export async function respond(request, options, manifest, state) {
 			// if this request came direct from the user, rather than
 			// via our own `fetch`, render a 404 page
 			if (state.depth === 0) {
+				// In local development, Chrome requests this file for its 'automatic workspace folders' feature,
+				// causing console spam. TODO should we instead respond with some actual data?
```

Comment:
could always delegate to https://github.com/ChromeDevTools/vite-plugin-devtools-json

Reply:
I sent https://github.com/vitejs/awesome-vite/pull/1030 to make it easier to find again if we need to track it down